### PR TITLE
SonarQube technical debt ratio

### PR DESCRIPTION
**Pull request by trask**
Add support for SonarQube's technical debt ratio.

Also implemented #430.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -544,15 +544,17 @@ cache(function(data, match, sendBadge, request) {
...
       var serverUrl = match[2];  // eg, `sonar.qatools.ru`.
       var buildType = match[3];  // eg, `ru.yandex.qatools.allure:allure-core:master`.
-      var format = match[4];
+      var metric = match[4];
+      var format = match[5];
+      var realMetric = metric === 'tech_debt' ? 'sqale_debt_ratio' : metric
       var apiUrl = scheme + '://' + serverUrl + '/api/resources?resource=' + buildType
-          + '&depth=0&metrics=coverage&includetrends=true';
-      var badgeData = getBadgeData('coverage', data);
+          + '&depth=0&metrics=' + realMetric + '&includetrends=true';
```
Comment (espadrine) [inline server.js]: Should we `encodeURIComponent(realMetric)`?

File: server.js [hunk 2]
```diff
@@ -544,15 +544,17 @@ cache(function(data, match, sendBadge, request) {
...
+      var badgeData = getBadgeData(escapeFormat(metric), data);
```
Comment (espadrine) [inline server.js]: Is there a reason why you used `escapeFormat()`? It was designed specifically to escape raw custom badge information from the URL. I don't think it makes sense here.

**Comment by espadrine** [comment]
Thanks for your contribution!

**Comment by trask** [comment]
@espadrine Thanks for the feedback, please see updates.

**Comment by espadrine** [comment]
Thanks a lot! Great work.

**Pull request merged**